.github/terraform/mainnet-dns.txt

```diff
@@ -0,0 +1 @@
+bafybeig7auw6b7svrctva7ozqbqgbf5ldireewaem73inktp6hgvcdtd5q.ipfs.dweb.link
```

Comment:
**issue (blocking)**: This should be prefix with `https://`, as it is a URL redirect not a CNAME